### PR TITLE
Modify font style and size

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
This PR is to add some of changes suggested by @timdansys as listed below:
- [x] improve the font size
- [x] Remove ansys-gold color from link hover 
- [x] Remove the bottom  border from breadcrumbs
- [x] Change color of switch button
- [x] Modify color of toctree
new-1
new-2

### CODE REVIEW

File: src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* Provided by the Sphinx base theme template at build time */
 @import "../basic.css";
-@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@600&display=swap');
+/*@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@600&display=swap');*/
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap');
```
Comment (RobPasMue) [inline src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:4]:
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap');
```

If we are commenting statements, I'd just remove them... Not a fan of commented source code.

**Review by RobPasMue — APPROVED** [review]
LGTM, though I am not experienced on the accessibility as the other reviewers. I'd leave them approve as well before merging

File: .gitignore [hunk 2]
```diff
@@ -120,6 +120,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.vscode
```
Comment (MaxJPRey) [inline .gitignore:123]: Thanks @Revathyvenugopal162 . That would be useful.

**Comment by MaxJPRey** [comment]
Thanks for adding the "underlying" when hovering over a main section at the top of the page.
image

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**